src/arvgvdevice.h

```diff
@@ -29,11 +29,33 @@
 
 G_BEGIN_DECLS
 
+#ifdef FAST_HEARTBEAT
+    #define        ARV_GV_DEVICE_GVCP_N_RETRIES_DEFAULT    3
+    #define        ARV_GV_DEVICE_GVCP_TIMEOUT_MS_DEFAULT   25
+    #define        ARV_GV_DEVICE_HEARTBEAT_PERIOD_US       50000
+    #define        ARV_GV_DEVICE_HEARTBEAT_RETRY_DELAY_US  1000
+    #define        ARV_GV_DEVICE_HEARTBEAT_RETRY_TIMEOUT_S 0.25
+#endif
+
+#ifndef	ARV_GV_DEVICE_GVCP_N_RETRIES_DEFAULT
```

Comment:
A single #if #else #endif sequence should be used here, instead of this multiple #ifndef #define #endif sequences.